### PR TITLE
GEODE-9567: Geode for Redis rename

### PR DESCRIPTION
**Pull request by ringles**
Rename "APIs Compatible with Redis" to "Geode for Redis". Affects module names, directory names, config and docs.

### CODE REVIEW

**Review by echobravopapa — APPROVED** [review]
@ringles LGTM, would you please fix your PR title to match standard format... e.g. GEODE-NNNN: description

**Review by DonalEvans — CHANGES_REQUESTED** [review]
Searching through the codebase, I still see a few instances of strings matching "compatiblewithredis" that might need to be updated, particularly in `geode-for-redis/README.md` and `geode-docs/developing/geode_for_redis.html.md.erb`.

File: geode-core/src/test/resources/org/apache/geode/management/internal/cli/commands/golden-help-offline.properties [hunk 1]
```diff
@@ -2572,7 +2572,7 @@ SYNTAX\n\
 \ \ \ \ [--lock-memory(=value)?] [--log-level=value] [--max-connections=value] [--max-heap=value]\n\
 \ \ \ \ [--max-message-count=value] [--max-threads=value] [--mcast-address=value] [--mcast-port=value]\n\
 \ \ \ \ [--memcached-port=value] [--memcached-protocol=value] [--memcached-bind-address=value]\n\
-\ \ \ \ [--compatible-with-redis-port=value] [--compatible-with-redis-bind-address=value] [--compatible-with-redis-password=value]\n\
+\ \ \ \ [--geode-for-redis-port=value] [--geode-for-redis-bind-address=value] [--geode-for-redis-password=value]\n\
```
Comment (jdeppe-pivotal) [inline geode-core/src/test/resources/org/apache/geode/management/internal/cli/commands/golden-help-offline.properties:2575]: I'm pretty sure this file can just be deleted - I don't see any references to it anywhere. Even before your changes it looks like it is referencing `geode-for-redis-password` which is no longer a valid param (I missed this reference when I removed that parameter).

File: geode-apis-compatible-with-redis/src/acceptanceTest/resources/0001-configure-redis-tests.patch [hunk 2]
```diff
@@ -1,2786 +0,0 @@
-diff --git a/tests/support/server.tcl b/tests/support/server.tcl
```
Comment (jdeppe-pivotal) [inline geode-apis-compatible-with-redis/src/acceptanceTest/resources/0001-configure-redis-tests.patch:1]: I think this file was accidentally removed.

**Review by jdeppe-pivotal — CHANGES_REQUESTED** [review]

File: geode-apis-compatible-with-redis/src/acceptanceTest/resources/0001-configure-redis-tests.patch (hunk 2 above)
Comment (ringles) [reply geode-apis-compatible-with-redis/src/acceptanceTest/resources/0001-configure-redis-tests.patch:1]: Oy. That's weird. I'll get it back in.

**Comment by jdeppe-pivotal** [comment]
Hmmm - there are exactly `666` files changed....

File: CODEOWNERS [hunk 3]
```diff
@@ -284,7 +284,7 @@ geode-assembly/**/resources/**                                    @boglesby @kir
 #-----------------------------------------------------------------
 # Redis API
 #-----------------------------------------------------------------
-geode-apis-compatible-with-redis/**                               @jdeppe-pivotal @nonbinaryprogrammer @ringles @upthewaterspout @DonalEvans @dschneider-pivotal @kirklund
+geode-for-redis/**                               @jdeppe-pivotal @nonbinaryprogrammer @ringles @upthewaterspout @DonalEvans @dschneider-pivotal @kirklund
```
Comment (onichols-pivotal) [inline CODEOWNERS:287]: please maintain indentation when changing CODEOWNERS

Comment (ringles) [reply CODEOWNERS:287]: Will do!

**Review by onichols-pivotal — APPROVED** [review]

**Review by dickcav — APPROVED** [review]

File: geode-docs/developing/geode_for_redis.html.md.erb [hunk 4]
```diff
@@ -49,20 +49,20 @@ Replace `<serverName>` with the name of your server.
...
 Replace `<locatorPort>` with your locator port.
 
-Replace `<compatibleWithRedisPort>` with the port that the <%=vars.product_name%> server
+Replace `<geodeForRedisPort>` with the port that the <%=vars.product_name%> server
  listens on for Redis commands. The typical port used with a Geode for Redis cluster is 6379.
 
-Replace `<compatibleWithRedisBindAddress>` with the address of the server host.
+Replace `<geodeForRedisBindAddress>` with the address of the server host.
 
 Replace `<compatibleWithWithRedisPassword>` with the password clients use to authenticate.
```
Comment (DonalEvans) [inline geode-docs/developing/geode_for_redis.html.md.erb:57]: This also needs to be updated. Easy to miss since it's a typo.

Comment (ringles) [reply geode-docs/developing/geode_for_redis.html.md.erb:57]: Good catch!

**Review by DonalEvans — APPROVED** [review]

**Review by jdeppe-pivotal — APPROVED** [review]

**Review by upthewaterspout — APPROVED** [review]

**Review by davebarnes97 — APPROVED** [review]
Approved with changes already applied.
(I edited the *.erb user guide sources and the README.md file to accommodate the name change.)

File: geode-for-redis/README.md [hunk 5]
```diff
@@ -97,41 +97,41 @@ not connected>
 
 ## <a name="security"></a>Security
 
-Security is implemented slightly differently to OSS Redis. Redis stores password information in plain text in the redis.conf file.     
+Security is implemented slightly differently from OSS Redis. Redis stores password information in plain text in the redis.conf file.     
 
-When using Apache Geode, to enable security, a Security Manager needs to be configured on the server(s). This Security Manager will authenticate `AUTH <password>` commands and `AUTH <username> <password>` commands. Users can set a custom `default` username using the `geode-compatible-with-redis-username` parameter. This username will be used when `AUTH <password>` commands are sent without a `<username>`. 
+When using Apache Geode, you must configure a Security Manager on the server(s) to enable security. The Security Manager authenticates `AUTH <password>` commands and `AUTH <username> <password>` commands. Users can set a custom `default` username using the `geode-geode-for-redis-username` parameter. The default username is used when `AUTH <password>` commands are sent without a `<username>`.
```
Comment (DonalEvans) [inline geode-for-redis/README.md:102]: Typo here, "geode-geode-for-redis-username" should be "geode-for-redis-username"

**Review by jinmeiliao — APPROVED** [review]

**Review by dschneider-pivotal — APPROVED** [review]

**Review by mhansonp — APPROVED** [review]

**Review by kirklund — APPROVED** [review]

File: geode-core/src/main/java/org/apache/geode/internal/cache/BucketRegion.java [hunk 6]
```diff
@@ -294,6 +294,16 @@ private void setEventSeqNum() {
     }
   }
 
+  @Override
```
[the remark this answers is not in the document]
Comment (onichols-pivotal) [reply geode-core/src/main/java/org/apache/geode/internal/cache/BucketRegion.java:297]: >  there are changes for the stats. 

@jchen21 the stat changes you are describing seem to have come from unrelated PR https://github.com/apache/geode/pull/6199

maybe there is just some issue with merge/rebase?

[the remark this answers is not in the document]
Comment (jchen21) [reply geode-core/src/main/java/org/apache/geode/internal/cache/BucketRegion.java:297]: It does look like merge/rebase issue.

**Review by jchen21 — APPROVED** [review]

**Pull request closed without merging**